From 956fedc7e0831126b9006efedad5519c14201c52 Mon Sep 17 00:00:00 2001
From: Jakub Wilk <jwilk@jwilk.net>
Date: Thu, 17 Feb 2022 00:26:41 +0100
Subject: [PATCH] pdf-backend: fix compat with Poppler > 22.02.

Fixes:

    pdf-backend.cc:152:3: error: no matching constructor for initialization of '::PDFDoc'
    /usr/include/poppler/PDFDoc.h:132:14: note: candidate constructor not viable: no known conversion from 'nullptr_t' to 'const std::optional<GooString>' for 2nd argument

https://cgit.freedesktop.org/poppler/poppler/commit/?id=4f2abd3efa1ee013
---
 pdf-backend.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pdf-backend.cc b/pdf-backend.cc
index c74053d..ba85da2 100644
--- a/pdf-backend.cc
+++ b/pdf-backend.cc
@@ -149,7 +149,7 @@ class unique_ptr_adapter
 };
 
 pdf::Document::Document(const std::string &file_name)
-: ::PDFDoc(unique_ptr_adapter<pdf::String>(new pdf::String(file_name.c_str())), nullptr, nullptr)
+: ::PDFDoc(unique_ptr_adapter<pdf::String>(new pdf::String(file_name.c_str())))
 {
   if (!this->isOk())
     throw LoadError();
